@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -218,6 +225,27 @@ export default {
       }
     },
 
+    buyAgain () {
+      const { localOrder } = this
+      if (localOrder.items) {
+        const { items } = localOrder
+        let countAddItems = 0
+        items.forEach(item => {
+          ecomCart.addItem(item)
+          countAddItems++
+          if (countAddItems === localOrder.items.length) {
```

Comment:
```suggestion
        items.forEach((item, i) => {
          ecomCart.addItem(item)
          if (i + 1 === localOrder.items.length) {
```

É só microperformance, mas não precisa e até simplifica sem este contador.